app/views/layouts/_header.html.erb

```diff
@@ -4,7 +4,9 @@
     <ul class="nav navbar-nav navbar-right">
       <% if current_user %>
         <ul>
-          <li><%= link_to "Create Business", new_business_path %></li>
+          <% if current_user.business.blank? || current_user.business.new_record? %>
```

Comment:
It's used for hiding create business link only when you have saved the record. When you are still in the form, then the link is still there.